packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -289,6 +305,8 @@ export class AtomicSearchBox implements InitializableComponent<Bindings> {
     }
   }
 
+  public disconnectedCallback: () => void = () => {};
```

Comment:
What is the point of this ? And why is it different from the one in atomic-commerce-search-box ?

`  public disconnectedCallback = () => {};
`